latest_downloads.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+if(!isset($_REQUEST['platform']))
+{
+	die("invalid request");
+}
+
+$replace = 1;
+switch($_REQUEST['platform'])
+{
+	case "win32x86":
+		$url = "https://s3.us-east-2.amazonaws.com/playbuilds/%s/Play-0.30-32.exe";
+		break;
+	case "win32x64":
+		$url = "https://s3.us-east-2.amazonaws.com/playbuilds/%s/Play-0.30-64.exe";
+		break;
+	case "macos":
+		$url = "https://s3.us-east-2.amazonaws.com/playbuilds/%s/Play.dmg";
+		break;
+	case "linux":
+		$url = "https://s3.us-east-2.amazonaws.com/playbuilds/%s/Play!-%s-x86_64.AppImage";
+		$replace = 2;
+		break;
+	case "android":
+		$url = "https://s3.us-east-2.amazonaws.com/playbuilds/%s/Play-release.apk";
+		break;
+	case "ios":
+		$url = "https://s3.us-east-2.amazonaws.com/playbuilds/%s/Play.ipa";
+		break;
+	break;
+	default:
+		die("invalid platform");
+}
+
+$content = file_get_contents("https://services.purei.org/api/builds");
+$json = json_decode($content);
+$hash = $json->commitHash;
+
+if($replace == 1)
```

Comment:
Could we use an actual replace function (with some kind of token like %COMMIT_HASH%) instead of having this if here?

Reply:
completely went blank on this XD